src/STAT_FrontEnd.h

```diff
@@ -926,15 +940,13 @@ class STAT_FrontEnd
 
         unsigned int launcherPid_;                          /*!< the PID of the job launcher */
         unsigned int nApplNodes_;                           /*!< the number of application nodes */
-        unsigned int proctabSize_;                          /*!< the size of the process table */
```

Comment:
The launchmon specific fields are moved into STAT_lmonFrontEnd.  

The process table got pulled into the lmon specific side because CTI avoids storing the full set of back-end processes on the front end to limit scaling problems.  That was the only real place where CTI and launchmon have a philosophical difference.    It makes the front end refactor a bigger lift than the back end one.